### DISCUSSION TITLE
Unexpected blank frame between trials

### DISCUSSION THREAD

**Discussion by cchateig** [jspsych/jsPsych#2663] [Q&A]
Hello,

I'm trying to build an experiment using jsPsych and cognition.run.
I've got two trials following each other, supposed to print the same fixation cross without any blank frame between them.
The fact is there is sometimes a blank frame and the cross sparkles.

I preload manually everything I can just before these two trials (an audio file for example which is played along with the cross at the beginning of the second trial). I use for both of the trials the plugin Psychophysics, and the fixation cross is defined as an object (obj_type : 'text'). I tried another version of the code in which the cross was an image and the trials were using the plugin HtmlKeyboardResponse. Same problem occured.

I have no idea why there is a delay before printing the fixation cross, and how I could get rid of that.

Does anyone here have an idea or have already encountered this problem ?

Thanks a lot,

Clara

**Comment by jodeleeuw** [comment]
Hi @cchateig,

Can you share a link to the cognition.run experiment and/or share the code for the trials that are showing the gap? Thanks!

**Reply by cchateig** [reply]
Hi ! Thank you for your reply.

This is a simplified version of my code, where the gap appears. It doesn't happen all the time.
The file 'klaxon_300ms.wav' is a 300ms audio file.

[psychophysics_scintillement_simplifie.txt](https://github.com/jspsych/jsPsych/files/8917177/psychophysics_scintillement_simplifie.txt)

**Reply by cchateig** [reply]
Hello again,

I tried to modify the psychophysics plugin in order to be able to have one trial instead of two without some errors I had, and it seems to work.

Thank you anyway for your help !
